components/NetworkStatusItem.qml

```diff
@@ -146,6 +147,46 @@ Rectangle {
                     }
                 }
             }
+
+            Text {
+                anchors.left: statusTextVal.right
+                anchors.leftMargin: 16
+                anchors.verticalCenter: parent.verticalCenter
+                color: refreshMouseArea.containsMouse ?  MoneroComponents.Style.dimmedFontColor : MoneroComponents.Style.defaultFontColor
+                font.family: FontAwesome.fontFamilySolid
+                font.pixelSize: 24
+                font.styleName: "Solid"
+                opacity: iconItem.opacity * (refreshMouseArea.visible ? 1 : 0.5)
+                text: FontAwesome.random
+                visible: item.connected && appWindow.walletMode < 2
```

Comment:
What about showing this button when booststrap-daemon is set to auto? This would also be useful for advanced mode users.

Reply:
Yep